src/resource_server/utils/signing_api.py

```diff
@@ -22,10 +22,10 @@ def check_signature(base_url, public_key: str, cert_key: str) -> Dict[str, str]:
         "public_key": public_key
     }
 
-    return _post_request(url, params)
+    return _post_request(url, params, headers)
 
-def _get_request(url: str) -> str:
-    response = requests.get(url)
+def _get_request(url: str, headers: Dict) -> str:
+    response = requests.get(url, headers=headers)
```

Comment:
passing all headers from the request unchecked is problematic, and may leak information or send unwanted headers / parameters.

Reply:
ok, I will check and just send the `Authorization`